src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/mgr-summary.pipe.ts

```diff
@@ -15,18 +15,22 @@ export class MgrSummaryPipe implements PipeTransform {
     }
 
     let activeCount = this.i18n('n/a');
-    const titleText = _.isUndefined(value.active_name)
+    const activetitleText = _.isUndefined(value.active_name)
       ? ''
       : `${this.i18n('active daemon')}: ${value.active_name}`;
-    if (titleText.length > 0) {
+    if (activetitleText.length > 0) {
       activeCount = '1';
     }
+
+    const standbytitleText = _.isUndefined(value.standbys)
```

Comment:
As said in my other comment I would prefer not to show a tooltip if we have nothing to show.

```suggestion
    const standbytitleText = !value.standbys // True for 0, Null and undefined
```